### PR TITLE
bump: Bump circleci orbs version and refactor

### PR DESCRIPTION
**Pull request by josemiguelmelo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -10,37 +11,28 @@ workflows:
...
           cmd: |
-            git clone --depth 1 git://github.com/codacy/codacy-plugins-test.git codacy-plugins-test
-            sbt ";set scalafmtUseIvy in ThisBuild := false;update"
-            (cd codacy-plugins-test && sbt update)
+            docker build -t codacy/alpine-jre-shellcheck -f Dockerfile .
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 scalafmt::test;
+                 test:scalafmt::test;
+                 sbt:scalafmt::test;
+                 test;
```
Comment (lolgab) [inline .circleci/config.yml]: We don't have tests in this repository

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.3.4
+sbt.version=1.3.5
```
Comment (lolgab) [inline project/build.properties]: There is `1.3.6` out

File: .circleci/config.yml (hunk 1 above)
Comment (josemiguelmelo) [reply .circleci/config.yml]: Removed it

File: project/build.properties (hunk 2 above)
Comment (josemiguelmelo) [reply project/build.properties]: Bumped

**Pull request merged**